content/en/tracing/trace_collection/custom_instrumentation/go/otel.md

```diff
@@ -164,6 +147,29 @@ ctx, span := t.Start(
 span.End()
 ```
 
+## Adding Span Events
+
+Add span events using the `AddEvent` API. Event name is a required first field, along with optional input for event `timestamp` and event `attributes`.
+In the following example, `oteltrace` is an alias for the go.opentelemetry.io/otel/trace package, this package must be imported in order to use this example.
+
+- **Name** [_required_]: A string representing the event's name.
+- **Attributes** [_optional_]: Zero or more key-value pairs with the following properties:
+  - The key must be a non-empty string.
+  - The value can be either:
+    - A primitive type: string, Boolean, or number.
+    - A homogeneous array of primitive type values (for example, an array of strings).
+  - Nested arrays and arrays containing elements of different data types are not allowed.
+- **Timestamp** [_optional_]: A UNIX timestamp representing the event's occurrence time, expects a `Time` object.
+  
+```go
+// Start a span.
+ctx, span := t.StartSpan(context.Background(), "span_name")
```

Comment:
```suggestion
ctx, span := tracer.StartSpan(context.Background(), "span_name")
```
I just think this makes the example clearer (...my initial span events example was not great 😄 )